config/prod.exs

```diff
@@ -14,7 +14,7 @@ use Mix.Config
 config :darkstone, Darkstone.Endpoint,
   http: [port: {:system, "PORT"}],
   url: [host: "example.com", port: 80],
-  cache_static_manifest: "priv/static/manifest.json"
+  cache_static_manifest: "priv/static/cahce_manifest.json"
 
```

Comment:
There is a typo here, `cahce` should be `cache`.

Also, why is `Darkstone.Endpoint` not changed to `DarkstoneWeb.Endpoint`?
A similar question can be asked for `Darkstone.Endpoint` in `test.exs`.